src/Tickets/Commerce/Gateways/PayPal/Client.php

```diff
@@ -499,10 +499,18 @@ public function create_order( array $units = [] ) {
 				$purchase_unit['items']               = $items;
 				$purchase_unit['amount']['breakdown'] = [
 					'item_total' => [
-						'value'         => Arr::get( $unit, 'value' ),
+						'value'         => $unit['item_value'] ?? $unit['value'],
 						'currency_code' => Arr::get( $unit, 'currency' ),
 					],
```

Comment:
Small suggestion, Since we had trouble understanding how the breakdown worked, I would link to the PayPal documentation section so that our future selves have any easy to find reference. https://developer.paypal.com/docs/api/payments/v2/#definition-amount_breakdown